operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -129,8 +129,10 @@ The Release Manager is able to look at the version we have deployed to Mattermos
 
 **Process for merging the `master` branch into the `cloud` branch**
 
-* The Cloud team opens PRs in webapp, server, redux, api-reference, and Enterprise branches to merge the `master` branch into the `cloud` branch \([example PR](https://github.com/mattermost/mattermost-server/pull/16838)\).
-* Devs should be aware of the dates when a `master` branch is merged into a `cloud` branch in order to be mindful of avoiding having incomplete features in a Cloud release, and to include bug fixes that we may want to include in a release.
+* Every two weeks on a Monday, the Server Team opens PRs in server, api-reference, and enterprise repositories to merge the `master` branch into the `cloud` branch \([example PR](https://github.com/mattermost/mattermost-server/pull/18916)\).
```

Comment:
```suggestion
* Every two weeks on a Monday, the Server Platform Team opens a PR in the server repository to merge the `master` branch into the `cloud` branch. \([example PR](https://github.com/mattermost/mattermost-server/pull/18916)\).
```

Actually we can follow the same process for `api-reference` and `enterprise` repositories as we do in the webapp repository.